docs/features/reading-progress-bar.md

```diff
@@ -0,0 +1,47 @@
+# Reading progress bar
+
+Feature ID: `readingProgressBar`
+
+## What it is?
+
+Add a reading progress bar to your pages to encourage visitors to scroll further down.
+
+## What it fits for?
+
+All article pages.
+
+## Potential impacts
+
+Higher **reading completion rate**, higher **ad impressions**, higher **ad revenue**.
```

Comment:
I don't understand why adding this feature will increase ad imps or ad revenue. Can you let me know even if it's offline?